### PR TITLE
Feat/keep detailed view opened

### LINKED ISSUE #800: [ResultList] "Zoom to" action launched from an detailed entry card closes the detailed entry card
### Description of the bug

In the result list, if we click on an entry, a detailed entry card is opened for that entry. In that card, if we click the "Zoom to" button, the map zooms to the entry, the result list gets updated and the detailed entry card is closed. This is arms the user experience, he can be frustrated to see the card his attention is focusing on disappearing.

### What was the error message shown?

```JavaScript
The detailed entry card disappears.
```

### Steps to reproduce

- Open the result list
- Click on one entry
- In the detailed entry card, click "zoom to"

### If the issue was noticed in the demo workspace, please specify the collection and the field. Otherwise, please link the 2 ARLAS config files.

This can be seen in the "ARLAS Earth Observation" dashboard of the demo.

### Which ARLAS component(s) are concerned by the bug?

Result list

### If you chose 'Other', please specify the part(s) of ARLAS concerned by the bug.

_No response_

### ARLAS version

23.1.0

### What browsers are you seeing the problem on?

Chrome

### If you chose 'Other', please specify the browser(s) concerned by the bug.

_No response_

### PR DESCRIPTION
**Pull request by MAudelGisaia**
close #800

### CODE REVIEW

File: src/app/components/arlas-wui-root/arlas-wui-root.component.ts [hunk 1]
```diff
@@ -589,6 +593,7 @@ export class ArlasWuiRootComponent implements OnInit, AfterViewInit, OnDestroy {
                 }
               }
             );
+            // this.retrieveOpenedGridItem();
```
Comment (QuCMGisaia) [inline src/app/components/arlas-wui-root/arlas-wui-root.component.ts:596]: To remove ?

File: src/app/components/arlas-wui-root/arlas-wui-root.component.ts [hunk 2]
```diff
@@ -792,6 +798,15 @@ export class ArlasWuiRootComponent implements OnInit, AfterViewInit, OnDestroy {
     this.mapSettings.openDialog(this.mapSettingsService);
   }
 
+  private retrieveOpenedGridItem(){
+    // wrong place because fire before result list opened
```
Comment (QuCMGisaia) [inline src/app/components/arlas-wui-root/arlas-wui-root.component.ts:802]: Does that mean that the method is not useful ?

File: src/app/components/arlas-wui-root/arlas-wui-root.component.ts [hunk 3]
```diff
@@ -1490,5 +1512,15 @@ export class ArlasWuiRootComponent implements OnInit, AfterViewInit, OnDestroy {
...
     }, 100);
   }
+
+  public storeItemIdentifier($event: Item) {
+    if($event.identifier) {
+      this._lastTileIdentifier = (<Item>$event).identifier;
+    }
+  }
+
+  public reactToUpdateList() {
```
Comment (QuCMGisaia) [inline src/app/components/arlas-wui-root/arlas-wui-root.component.ts:1522]: onListUpdated ?

File: src/app/components/arlas-wui-root/arlas-wui-root.component.ts (hunk 2 above)
Comment (MAudelGisaia) [reply src/app/components/arlas-wui-root/arlas-wui-root.component.ts:802]: no its an old comment. I ll remove it

**Review by sebbousquet — APPROVED** [review]

**Review by MohamedHamouGisaia — APPROVED** [review]

File: src/app/components/arlas-wui-root/arlas-wui-root.component.ts [hunk 4]
```diff
@@ -799,11 +798,12 @@ export class ArlasWuiRootComponent implements OnInit, AfterViewInit, OnDestroy {
   }
 
   private retrieveOpenedGridItem(){
```
Comment (QuCMGisaia) [inline src/app/components/arlas-wui-root/arlas-wui-root.component.ts:800]: The accessibility should be protected, otherwise I don't think it will work

**Review by QuCMGisaia — APPROVED** [review]

**Pull request merged**